### DISCUSSION TITLE
VanillaPlus v1.1

### DISCUSSION THREAD

**Discussion by MindustryDagger** [MindustryDagger/VanillaPlus#20] [Ideas]
## What's Changed
* rebalance update by @MindustryDagger in https://github.com/MindustryDagger/VanillaPlus/pull/17
* Continue work with rebalance by @MindustryDagger in https://github.com/MindustryDagger/VanillaPlus/pull/19

**Full Changelog**: https://github.com/MindustryDagger/VanillaPlus/compare/v1.0...v1.1

-rebalance and Eng translations

<hr /><em>This discussion was created from the release <a href='https://github.com/MindustryDagger/VanillaPlus/releases/tag/v1.1'>VanillaPlus v1.1</a>.</em>

**Comment by TeslaX1983** [comment]
хеё бро. Я пишу по поводу того что мод немного сломан, даже в стиме. Он попросту не работает и выдает ошибку, версия к тому же 157.4, нам с дургом не хватает этого мода, мы не знаем что делать, потому что ошибка в названии сектора и в танковой фабрике 2
